definitions/private-law/json/CaseEventToFields/SubmitAndPay/CaseEventToFields.json

```diff
@@ -145,21 +101,10 @@
     "DisplayContext": "MANDATORY",
     "PageID": 2,
     "PageDisplayOrder": 2,
-    "PageFieldDisplayOrder": 13,
+    "PageFieldDisplayOrder": 9,
     "ShowSummaryChangeOption": "Yes",
     "FieldShowCondition": "submitAndPayDownloadApplicationLink=\"DO_NOT_SHOW\""
   },
-  {
-    "LiveFrom": "02/12/2021",
-    "CaseTypeID": "PRLAPPS",
-    "CaseEventID": "submitAndPay",
-    "CaseFieldID": "submitAndPayDownloadApplicationWelshLink",
-    "PageFieldDisplayOrder": 14,
-    "DisplayContext": "READONLY",
-    "PageID": 2,
-    "PageDisplayOrder": 2,
-    "FieldShowCondition": "payAgreeStatement=\"agree\" AND isEngDocGen=\"No\""
-  },
```

Comment:
why removed this one?

Reply:
This was part of the old declaration which has been changed in the screens making this field no longer required here.

Reply:
is it not common for solicitor payment journey?